ctapipe/core/traits.py

```diff
@@ -48,13 +49,28 @@
     "TelescopeParameter",
     "FloatTelescopeParameter",
     "IntTelescopeParameter",
+    "DateTime"
 ]
 
 import logging
 
 logger = logging.getLogger(__name__)
 
 
+class DateTime(TraitType):
+    """ A trait representing a point in Time, as understood by `astropy.time`"""
+    def validate(self, obj, value):
+        """ try to parse and return an ISO time string """
+        try:
+            the_time = Time(value)
+            return the_time.iso
```

Comment:
Why return the string and not the time instance? Now if one wants to use it, one has to convert it back to a time again.

There might also be a confusion with datetime of the standardlib.

Reply:
I'll change it to Timestamp.

Reply:
In the end I used `AstroTime` since it's too easy to have import problems with other `Time` objects, and this way it's clearer it's an Astropy Time.

Reply:
Not really. AStropy internally converts into two jd floats in a specific scale. It's only the default representation that sticks.

You could set it consistently independent of input format by using:

```python
>>> t  = Time('2020-01-20 20:00:00')
>>> print(t)
2020-01-20 20:00:00.000
>>> t.format = 'isot'
>>> print(t)
2020-01-20T20:00:00.000
```

Reply:
ah, good - I'll set it to ISO then, and that should work